src/templates/static/metatrader/download.jsx

```diff
@@ -1,108 +1,228 @@
+        children,
+        os,
+    }
+) =>
+    (
+        <a
+            className={!badge ? 'button' : undefined}
+            href={os ? `https://download.mql5.com/cdn/mobile/mt5/${os}?server=Binary.com-Server` : href}
+            download={download || undefined}
+            target={target || undefined}
+        >
+            <span className={badge || undefined}>{children}</span>
+        </a>
+    );
 
+const DesktopDownloadBlock = (
+    {
+        badge,
+        download,
+        href,
+        image,
+        os,
+        target,
+        title,
+        id,
+    }
+) => (
+    <div
+        className='gr-12 gr-12-m gr-padding-10 invisible download-block'
+        id={id}
+    >
+        <img
+            className='gr-12 gr-centered gr-padding-20 mt-desktop-screen-app'
+            src={it.url_for(`images/pages/metatrader/icons/${image}`)}
+        />
         <div className='gr-padding-20'>
-            <h3>{it.L('For Desktop') }</h3>
-            <p>{it.L('Download MT5 for your desktop or laptop to access the powerful tools and features enjoyed by millions of traders.') }</p>
-            <div className='gr-row'>
-                <DownloadBlock
-                    title={it.L('Windows')}
-                    desc={it.L('Windows 7 or later')}
-                    image='windows'
-                    href='https://s3.amazonaws.com/binary-mt5/binarycom_mt5.exe'
-                />
-                <DownloadBlock
-                    title={it.L('MacOS')}
-                    desc={it.L('All versions of MacOS')}
-                    image='apple'
-                    href='https://s3.amazonaws.com/binary-mt5/binary-mt5.dmg'
-                    download='true'
-                />
-                <DownloadBlock
-                    title={it.L('Linux')}
-                    desc={it.L('All versions of Linux')}
-                    image='linux'
-                    href='https://www.metatrader5.com/en/terminal/help/start_advanced/install_linux'
-                    target='_blank'
-                />
-            </div>
-            <p className='foot-note gr-padding-20'>{it.L('The MetaTrader platform will no longer support Windows XP, Windows 2003, and Windows Vista after 01 October, 2017.') }</p>
+            <DownloadButton
+                badge={badge}
+                href={href}
+                download={download}
+                os={os}
+                target={target}
+            >
+                {it.L(`Download MT5 for ${title}`)}
+            </DownloadButton>
         </div>
+    </div>
+);
 
-        <SeparatorLine no_wrapper />
+const Download = () => {
+    const windows_link = `<a data-type='alt-link' data-target='windows' id='enable-windows-alt-link'>${it.L('Windows')}</a>`;
+    const linux_link = `<a data-type='alt-link' data-target='linux' id='enable-linux-alt-link'>${it.L('Linux')}</a>`;;
+    const mac_link = `<a data-type='alt-link' data-target='mac' id='enable-mac-alt-link'>${it.L('Mac')}</a>`;;
```

Comment:
could you align these by `=`

Reply:
I'm getting confused by [this](https://github.com/binary-com/binary-static/pull/4408#pullrequestreview-160991464)

Reply:
for variables that are grouped together we usually align by `=`, if they are grouped by themselves then there's no need for extra whitespace